dcpy/connectors/esri/arcgis_feature_server.py

```diff
@@ -0,0 +1,52 @@
+from enum import StrEnum
+import requests
+from pydantic import BaseModel
+from typing import cast
+
+
+class Server(StrEnum):
+    nys_clearinghouse = "nys_clearinghouse"
+
+    @property
+    def id(self):
+        match self:
```

Comment:
This is silly at the moment, I'll just make a dict and non-class function. Will probably play nicer with pydantic anyways